GitUI/Theming/ThemeModule.cs

```diff
@@ -26,23 +26,6 @@ public static void Load()
             Win32ThemeHooks.ThemeSettings = Settings;
         }
 
-        private static void InstallHooks(Theme theme)
```

Comment:
This function (and probably the other) had to be disabled in the .NET5 merge with `#if SUPPORT_THEMES`, as there is no solution for EasyHook yet.
I really hope that we can restore this functionality soon, so I do not want to loose theming.

Reply:
Why don't we added TODO or just a comment in that case? I strongly recommend adding comments to the preserved code for future purposes, so people start contributing tomorrow will not lose their efforts.

Reply:
The .NET5 is quite a big change that RussKie has spent a lot of time on, rebaseing many times. 
Great work but he had to merge to get going and everything is not perfect. (there are some tasks in the PR). Be easy on RussKie, he spends a lot of time on GE.

The usage of the functions are marked.
I would vote for a PR setting the `#if` around these functions (assuming all are for theming).

Reply:
I couldn't see any #if blocks for the functions I removed. BTW, since this PR is drafted, we can ignore the causes.

Reply:
> Theme-related functionality must be commented out with `#if SUPPORT_THEMES`

Yes. That's my point. If there is no `#if ` block for the functions, it's hard to predict the future purposes especially for the new contributors.